lib/lte_link_control/lte_lc.c

```diff
@@ -43,6 +43,23 @@ LOG_MODULE_REGISTER(lte_lc, CONFIG_LTE_LINK_CONTROL_LOG_LEVEL);
 		LTE_LC_SYSTEM_MODE_LTEM_NBIOT_GPS		: \
 	LTE_LC_SYSTEM_MODE_DEFAULT)
 
+/* Internal enums */
+
+enum feaconf_oper {
+	FEACONF_OPER_WRITE = 0,
+	FEACONF_OPER_READ  = 1,
+	FEACONF_OPER_LIST  = 2
+};
+
+enum feaconf_feat {
+	FEACONF_FEAT_PROPSM = 0
```

Comment:
True, I'll change it.

Reply:
I'm even thinking if some places should use proprietary. Especially function names but why not all. The only reason (I don't count 5-6 extra characters as too bad thing) not to change is that PROPRIPSM is aligned with AT command.

Reply:
We can use "proprietary" as well, it's not that much longer. We're anyway not aligned with the AT command because the feature is controlled with %FEACONF command now. We can further discuss the naming offline.

Reply:
Updated LTE LC naming.